hack/test-gce/nodeagent_install.sh

```diff
@@ -0,0 +1,282 @@
+#! /bin/bash
+
+set -e
+
+#Golang version
+GO_VERSION=${GO_VERSION:-"1.18.7"}
+
+pushd $HOME
+echo -e "## SETTING UP THE HOSTNAME NODEAGENT-A\n"
+sudo hostnamectl set-hostname nodeagent-a
+echo -e "## DISABLING FIREWALL\n"
+sudo ufw disable
+sudo swapoff -a
+
+basic_install() {
+    echo -e "## INSTALL BASIC TOOL"
+    sudo apt-get -y update
+    sudo apt -y install build-essential
+    sudo apt -y install curl
+    sudo apt-get -y install vim
+    echo -e "## DONE BASIC TOOL\n"
+}
+
+
+docker_install(){
+   sudo apt-get update -y > /dev/null 2>&1
+   if [ "$(which docker)" != "" ] > /dev/null 2>&1
+    then
+       echo -e "## DOCKER IS ALREADY INSTALLED\n"
+    else
+       echo -e "##INSTALLING DOCKER"
+       sudo curl -fsSL https://get.docker.com -o get-docker.sh
+       sudo sh get-docker.sh > /dev/null 2>&1
+       sudo chmod o+rw /var/run/docker.sock; 
+       ls -al /var/run/docker.sock
+       echo -e "## DOCKER INSTALLED\n"
+   fi
+}
+
+
+runtimes_setup(){
+    echo -e "## Start Setup and Configuration\n"
+    daemon_install
+    config_runtimes
+    crictl_install
+    cni_install
+    cni_config
+    runsc_install
+    kata_install
+
+    systemctl restart docker
+    sleep 5
+}
+
+daemon_install() {
+    sudo apt-get -y update > /dev/null 2>&1
+    echo -e "## Write daeman.json File.\n"
+    sudo touch /etc/docker/daemon.json
+    sudo chmod 777 /etc/docker/daemon.json
+cat << EOF | sudo tee /etc/docker/daemon.json
+{
+    "runtimes": {
+        "quark": {
+            "path": "/usr/local/bin/quark"
+        },
+        "quark_d": {
+            "path": "/usr/local/bin/quark_d"
+        },
+        "runsc": {
+            "path": "/usr/local/bin/runsc"
+        },
+        "kata-runtime": {
+            "path": "/usr/bin/kata-runtime"
+        }
```

Comment:
we only need runc.

Reply:
this is our basic setting for test environment setting. not any affect test. default is runc